packages/frontend/lib/wagmiClient.ts

```diff
@@ -25,7 +24,7 @@ export const supportedChains: Chain[] = allChains.filter((chain) =>
   env.supportedChains.includes(chain.id)
 )
 
-const { provider } = configureChains(
+export const { chains, provider } = configureChains(
   Array.from(new Set([defaultChain, chain.mainnet, ...supportedChains])),
```

Comment:
TL;DR Opened an issue here that describes everything: https://github.com/rainbow-me/rainbowkit/issues/631. I would wait for their reply before merging (they are generally super fast)

Reply:
Update: They do not seem to answer. As I don't want this to be blocking us, I think we can accept it the way you suggested for this PR and change when they reply :)

What do you think regarding the other two points I suggested?

Reply:
I agree with both and with moving forward. I'll work on it this afternoon.